### PR TITLE
Added documentation for using crawlera with CasperJS and PhantomJS

### PR DESCRIPTION
**Pull request by Granitas**

### CODE REVIEW

**Comment by qrilka** [comment]
@Granitas wouldn't it be cleaner to fix `crawlera.rst` formatting in a separate commit or maybe even PR?

File: _static/crawlera-casperjs.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// Might need this if --proxy argument doesn't work
+//phantom.casperPath = '<PATH TO CASPERJS>';
+//phantom.injectJs(phantom.casperPath +'/bin/bootstrap.js');
+
+var casper = require('casper').create();
+// 1. session authentication
+casper.start();
+casper.setHttpAuth('<API key>', '');  // set up authentication for entire session
+casper.start('http://paygo.crawlera.com:8010/fetch?url=http://wtfismyip.com/text', function(response) {
```
Comment (qrilka) [inline _static/crawlera-casperjs.js]: URL params should be urlencoded

**Comment by qrilka** [comment]
BTW @Granitas is it possible to use Crawlera "normal" proxy mode or it is a problem to do that with casperjs?

**Comment by Granitas** [comment]
@qrilka 
You mean in code? AFAIK in code there are no in build tools for proxy. You could use some external library like https://github.com/SpookyJS/SpookyJS to achieve that, do you think it is worth mentioning?

**Comment by Granitas** [comment]
Restructured to include SpookyJS, what do you think @qrilka ?

**Pull request merged**